openslides/core/static/js/core/base.js

```diff
@@ -106,10 +106,14 @@ angular.module('OpenSlidesApp.core', [
             // get detected browser language code
             getBrowserLanguage: function () {
                 var lang = navigator.language || navigator.userLanguage;
-                if (lang.indexOf('-') !== -1)
-                    lang = lang.split('-')[0];
-                if (lang.indexOf('_') !== -1)
-                    lang = lang.split('_')[0];
+                try {
```

Comment:
I do not understand why we have to use a try-catch here. Which line in the old code raises a typeError in which cases?

The expected value of `navigator.language` is something like `de-DE` or `en_US` or thelike or just `undefinded`, isn't it?. So we should implement at first a check if the language attribute is set (or just undefined), then we can parse the language string.